content/ja/docs/concepts/signals/traces.md

```diff
@@ -2,8 +2,7 @@
 title: トレース
 weight: 1
 description: アプリケーションを通過するリクエストの経路
-default_lang_commit: 9b5e318
-cSpell:ignore: Guten
+default_lang_commit: b7e40731390448f604897ded62cff8abd3505430
```

Comment:
As you can see from https://github.com/open-telemetry/opentelemetry.io/compare/9b5e318...b7e40731390448f604897ded62cff8abd3505430, the cSpell entry was not removed, it was moved:

> <img width="444" alt="image" src="https://github.com/user-attachments/assets/8915ac6a-5ef8-4194-ae96-6935dbf618b9" />

I know that we do not currently spell-check Japanese pages, but we might at some point, and it's best to keep the same front matter fields as the `en` version of the page (unless they break the build).

```suggestion
default_lang_commit: b7e40731390448f604897ded62cff8abd3505430
cSpell:ignore: Guten
```

Reply:
@chalin 

Thank you for your checking.
I fixed a bellow commit.

https://github.com/open-telemetry/opentelemetry.io/pull/6293/commits/6e2a82d19f4c6b8da778d173c798df21c3ddb198